Add wiki pages for identity v1.1 (#1414)
iotaledger/iota-wiki: docs/build/identity.rs/1.1/docs/explanations/verifiable-credentials.mdx

like a driver's license, passports, or a person's age.
Verifiable credentials (VCs) are statements (e.g., Alice has a driver's license)
that can be cryptographically verified by a third party, either online or in person.
Additionally, the holder of the VC decides what is shared and who it is shared with.

There are several types of actors that play different roles in a verifiable credential system.
We'll start with a common example of how things work in the world today using physical credentials and centralized databases,
and outline the roles that various entities play in the Verifiable Credential system.

## Example - Passport Issuance

A government (the _issuer_) issues a passport asserting citizenship (the _Verifiable Credential_) to Alice (the _subject_ and _Holder_),
and writes the information to a database (the _Verifiable Data Registry_).
When crossing the border, Alice (the _Holder_) presents her passport to a border agent (the _Verifier_),
who can verify that Alice (the _subject_) is indeed a citizen.

**Subject:** An entity about which claims are made – Alice (the _subject_) is a citizen of this country.

> **Holder:** An entity which possesses verifiable credentials – Alice (the _Holder_) possesses the passport (the _VC_).

Reviewer: ```suggestion
**Holder:** Any entity with a verifiable credential – Alice (the _Holder_) possesses the passport (the _VC_).
```

**Issuer:** An entity which asserts claims about a subject – The governing body (the _issuer_), which is trusted, issues Alice a passport.

> **Verifier:** An entity which check's if the VC a holder presents is legitimate – The border agent (the _Verifier_) trusts the government (the _issuer_) which issued Alice her passport, and validates that Alice (the _subject_) is a citizen.

Reviewer: ```suggestion
**Verifier:** An entity which checks if the VC a holder presents is legitimate – The border agent (the _Verifier_) trusts the government (the _issuer_) which issued Alice her passport and validates that Alice (the _subject_) is a citizen.
```

:::note

See the [Verifiable Credentials Data Model 1.0 Specification](https://w3c.github.io/vc-data-model/) for more information.

:::

## Verifiable Credentials in IOTA

> In the IOTA Identity framework, instead of a physical passport being given to Alice with the information being written

Reviewer: ```suggestion
In the IOTA Identity framework, instead of a physical passport being given to Alice and its information written
```

into a centralized database owned by the government, Alice receives a digital verifiable credential,
and the information required for verification in the future is written to the Tangle.

At a high level, the creation and verification of a VC on IOTA works as follows:

> The first step is to create a verifiable credential which requires the subject (Alice) and issuer (the government) to

Reviewer: ```suggestion
The first step is to create a verifiable credential that requires the subject (Alice) and issuer (the government) to
```

> have [DIDs](./decentralized-identifiers.mdx) published on the Tangle, and a set of statements being asserted (that Alice has a passport).

Reviewer: ```suggestion
have [DIDs](./decentralized-identifiers.mdx) published on the Tangle and a set of statements being asserted (that Alice has a passport).
```

The issuer signs the credential with their private key and publishes the public key to the Tangle.

> Once the issuer is confident that the credential satisfies its expectation,

Reviewer: ```suggestion
Once the issuer is confident that the credential satisfies its expectations,
```

the credential is stored and transmitted to the subject in a secure manner (off-chain).

Validation is performed by looking up the issuer's public key on the Tangle,
the holder proving ownership of their DID to the verifier (evidence),

> and validating that the credential has indeed been signed by the issuing party.

Reviewer: ```suggestion
and validating that the issuing party has indeed signed the credential.
```
